Redoublet-frontend/src/components/Pixi/PixiMultiplayerComponent.vue

```diff
@@ -394,29 +395,67 @@ function download(text) {
 
 /**Gets the DDS info from the DDS server, then creates a clean table for the make-able contracts. */
 const analyse = async() => {
-    const response = await fetch('https://redoublet-dds-mwy2coqowq-ez.a.run.app/DDS', {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json'
-        },
-        body: JSON.stringify({
-            hand: DDSDeal,
-            contract: "",
-            declarer: "",
-        })
+
+const DealerParser = await fetch(`${backendUrl}/api/DealerParserString`, {
+    method: 'POST',
+    headers: {
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(currentGamestate.value.dealer)
```

Comment:
```suggestion
    body: JSON.stringify(currentGamestate.dealer)
```

in multiplayer is de gamestate geen ref, dus heeft het geen value property.